docs/installation/installing-with-npm.md

```diff
@@ -127,9 +127,13 @@ Include the `node_modules/govuk-frontend/all.js` script on your page. You might
 
 To initialise all components, use the `initAll` function.
 
+JavaScript in GOV.UK Frontend requires HTML to be first parsed by the browser, otherwise it will block render. This may stop components from functioning or displaying correctly. To avoid this, make sure you include the script before the closing `</body>` tag.
```

Comment:
As I understand it, the main reason it's important for users to include it at the end of their document is because we're not wrapping our init up in a `documentReady`, so it'll only affect elements that are already part of the DOM when the script is run?

Do we want to be clearer about this?

At the minute, it sounds like the most important reason is that it affects performance and that it _might_ result in your components not working, but actually we _know_ that it _will_ result in components not being initialised, and that it _might_ impact performance.

Reply:
reworded to make it more explicit